src/Domain.jl

```diff
@@ -83,14 +83,19 @@ function ConstantTPDomain(;phase::E2,initialconds::Dict{X,X2},constantspecies::A
     C = P/(R*T)
     V = N*R*T/P
     kfs,krevs = getkfkrevs(phase=phase,T=T,P=P,C=C,N=N,ns=ns,Gs=Gs,diffs=diffs,V=V)
+    kfsp = deepcopy(kfs)
+    for ind in efficiencyinds
+        kfsp[ind] = 1.0
```

Comment:
It's related to the definition of the parameters...and it results in some annoyances later. So if you suppose you're simulating with constant rate coefficients it's easiest (and more important from a speed perspective because it's already fast to do an evaluation) to perturb the values of the rate coefficients. However if you're recomputing the rate coefficients the most logical way to handle parameters is to multiply in a perturbation factor. The problem is during a ConstantTPDomain IdealGas simulation rate coefficients involving efficiencies need recomputed each iteration the other coefficients don't, so the parameters are adjusted accordingly.